rascal-vscode-extension/src/dap/RascalDebugClient.ts

```diff
@@ -77,13 +77,11 @@ export class RascalDebugClient {
 
     registerDebugServerPort(processID: number, serverPort: number){
         this.debugSocketServersPorts.set(processID, serverPort);
+        commands.executeCommand('rascalmpl.updateDebugView');
```

Comment:
Hmm this is a bit strange, I think we should use some kind of event system here, where the debug view just subscribes to the RascalDebugClient for updates. vscode library already has a buildin set of classes for this.

Reply:
Thanks for the suggestion